content/news/playtest-20200426.md

```diff
@@ -0,0 +1,68 @@
+<div style="border-radius: 4px; background-color: #272d2c; padding: 5px">
+<div style="margin: -10px 5px" markdown="1">
+
+
+Another weekend, another playtest! To help a small number of players who are experiencing problems with poor GPU drivers, we are today releasing [playtest-20200429](/download/) which restores the ability to use OpenGL 2.1.
+
+OpenRA will continue to prefer OpenGL 3.2+ (or OpenGL ES 3.0+), but a new option in the settings menu allows you to switch back to the legacy rendering API if you experience glitches or performance issues.
+
+<div style="text-align:center" markdown="1">
+<img src="/images/news/20200426-glprofile.png" alt="OpenGL Profile setting." width="650px">
+
+Support for legacy OpenGL 2.1 drivers has been restored.
+</div>
+
+This playtest also includes a scripting fix for two RA campaign missions, but no other gameplay changes. We believe that all of the important issues have now been resolved, so plan a final release in the near future if no issues are reported with the updated graphics support.
+
+</div>
+</div>
+
+For more information, see the previous news posts for [playtest-20200418](/news/playtest-20200418/), [playtest-20200329](/news/playtest-20200329/), and the [original playtest-20200303 announcement](/news/playtest-20200303/) post is included below:
```

Comment:
`which is included below`?

Reply:
or just `included below` without the `is`.